extension/src/engine/ws-client.ts

```diff
@@ -3,8 +3,32 @@ import { WebIdeClientJsonRpc } from './api/jsonrpc';
 import * as vscode from 'vscode';
 import { animationSettings, handleOpenProcessEditor, openEditor } from './animation';
 import { IWebSocket, WebSocketMessageReader, WebSocketMessageWriter } from 'vscode-ws-jsonrpc';
+import { LogClientJsonRpc } from '@axonivy/log-view-core';
 
 export const WebSocketClientProvider = (webSocketUrl: URL) => {
+  const runtimeLogWebSocket = new WebSocket(new URL('ivy-runtime-log-lsp', webSocketUrl));
+  runtimeLogWebSocket.onopen = () => {
+    const connection = toSocketConnection(runtimeLogWebSocket);
+    const outputChannel = vscode.window.createOutputChannel('Axon Ivy Runtime Log');
+
+    function appendLog(entry: { timestamp: string; level: string; project: string; message: string; stacktrace?: string }) {
```

Comment:
make a const instead of function similar to `const toSocketConnection`, and move it to the outer scope to make it more readable. nesting stuff makes it hard to read.